scripts/build.pbs

```diff
@@ -0,0 +1,7 @@
+#PBS -l nodes=1:ppn=16
+#PBS -l walltime=00:10:00
+
+#export PATH=/opt/sw/java/jdk-11.0.2/bin:$PATH
+export PATH=/opt/sw/java/java-se-8u41-ri/bin:$PATH
```

Comment:
user-specific paths in this script (and the next). I suppose not a big issue

Reply:
Oh yes, well it's the recommended way on the cluster unfortunately. We're encouraged to do this rather than request changes to the base system.